modules/localgov_login_redirect/localgov_login_redirect.module

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @file
+ * Hooks for LocalGov Login Page module.
+ */
+
+use Drupal\Core\Session\AccountInterface;
+use Drupal\Core\Url;
+
+/**
+ * Implements hook_user_login().
+ */
+function localgov_login_redirect_user_login(AccountInterface $account) {
```

Comment:
Is this going to be too blunt? If I look at https://git.drupalcode.org/project/login_destination/-/blob/8.x-2.x/login_destination.module#L46 they do stuff like check that it's not a one time login link to reset the password. This is in part because that module lets you configure these things, but also tells me it needs to be accounted for.

Reply:
Good catch - we don't want to redirect people when they're resetting passwords.